integration/discovery/discovery_test.go

```diff
@@ -634,6 +654,97 @@ var _ = Describe("DiscoveryService", func() {
 		Eventually(sess, network.EventuallyTimeout).Should(gexec.Exit(1))
 		Expect(sess.Err).To(gbytes.Say(`access denied`))
 	})
+
+	When("a peer is bootstrapped from a snapshot", func() {
+		var (
+			peerProcess ifrit.Process
+			snapshotDir string
+		)
+
+		BeforeEach(func() {
+			By("installing and instantiating chaincode")
+			chaincode := nwo.Chaincode{
+				Name:    "mycc",
+				Version: "1.0",
+				Path:    "github.com/hyperledger/fabric/integration/chaincode/simple/cmd",
+				Ctor:    `{"Args":["init","a","100","b","200"]}`,
+				Policy:  `OR (AND ('Org1MSP.member','Org2MSP.member'), AND ('Org1MSP.member','Org3MSP.member'), AND ('Org2MSP.member','Org3MSP.member'))`,
+			}
+			nwo.DeployChaincodeLegacy(network, "testchannel", orderer, chaincode, org1Peer0)
+
```

Comment:
This shouldn't be required for this test... no?

Reply:
Not required

Reply:
Removed